source/ast/expressions/AssertionExpr.cpp

```diff
@@ -417,6 +490,71 @@ void SequenceRange::serializeTo(ASTSerializer& serializer) const {
         serializer.write("max", "$"sv);
 }
 
+bool SequenceRange::operator!=(const SequenceRange& right) const {
```

Comment:
I think the logic here could just be defaulted? You're just comparing field by field right? So `operator==(const SourceRange&) = default;` should have the correct behavior?

Reply:
it was removed in terms of fixing a previous comment